### DISCUSSION TITLE
Support for interval models

### DISCUSSION THREAD

**Discussion by sjunges** [moves-rwth/storm#762] [Ideas]
With https://github.com/moves-rwth/storm/pull/409, storm has now preliminary support for intervals. This is by no means stable. Some important features that we are still missing are:

 - [ ] Explicit support for solving on DTMCs -- the DTMCs must be declared as MDPs
 - [ ] Support for rewards (this has been partially added in #627 and #741)
 - [ ] Bounded reachability properties
 - [ ] Topological VI
 - [ ] Policy extraction
 - [ ] Convenience functions to remove uncertainty (e.g., by taking the 'center').
 - [ ] support for robustness in formulas
 - [x] Generating interval models from PRISM/JANI
 - [ ] Exact intervals

**Comment by volkm** [comment]
I think at least support for parsing interval models from Prism/JANI would be nice to have soonish.
